test/meadow/ingest/sheets_test.exs

```diff
@@ -149,4 +150,27 @@ defmodule Meadow.Ingest.SheetsTest do
       assert Sheets.work_count(ingest_sheet) == 30
     end
   end
+
+  describe "fix stuck ingest sheet" do
+    @fixture "test/fixtures/ingest_sheet.csv"
+
+    setup do
```

Comment:
This test setup ended up being pretty hard to figure out, any suggestions to improve this are welcome!